src/java/yaw/engine/mesh/Material.java

```diff
@@ -27,6 +27,14 @@ public class Material {
     private Texture map_Ns;
     private float opacity;
 
+    // PBR

```

Comment:
I think there should be 3 distinct classes :   a `Material` abstract class with common properties and two subclasses : `MaterialADS` and `MaterialPBR`